fs-repo-11-to-12/migration/swapper.go

```diff
@@ -19,6 +21,32 @@ var SyncSize uint64 = 20 * 1024 * 1024 // 20MiB
 // migration.
 var NWorkers int = 4
 
+func init() {
+	workerEnvVar := "IPFS_FS_MIGRATION_11_TO_12_NWORKERS"
+	syncSizeEnvVar := "IPFS_FS_MIGRATION_11_TO_12_SYNC_SIZE"
+	if nworkersStr, nworkerInEnv := os.LookupEnv(workerEnvVar); nworkerInEnv {
+		nworkers, err := strconv.Atoi(nworkersStr)
+		if err != nil {
+			panic(err)
+		}
+		if nworkers < 1 {
+			panic("number of workers must be at least 1")
+		}
+		NWorkers = nworkers
+	}
+
+	if syncSizeStr, syncSizeInEnv := os.LookupEnv(syncSizeEnvVar); syncSizeInEnv {
+		syncSize, err := strconv.ParseUint(syncSizeStr, 10, 64)
+		if err != nil {
+			panic(err)
+		}
+		if syncSize < 1 {
+			panic("sync size bytes must be at least 1")
+		}
```

Comment:
Should we set this higher (e.g. 1MB minimum) just so people don't accidentally think 1 == 1MB or something?

Similarly is there any use for setting `0` to mean only syncing at the end of the migration?

Reply:
let's rename the env var to sync_size_bytes?